Bugsnag/KSCrash/Source/KSCrash/Recording/BSG_KSCrashC.c

```diff
@@ -90,6 +93,14 @@ void bsg_kscrash_i_onCrash(BSG_KSCrash_Context *context) {
 #pragma mark - API -
 // ============================================================================
 
+void bsg_kscrash_init(void) {
+    if (!bsg_g_initialised) {
+        bsg_g_initialised = true;
+        memset(&bsg_g_crashReportContext, 0, sizeof(bsg_g_crashReportContext));
```

Comment:
A call to `memset` is not required, `bsg_g_crashReportContext` is already initialized to 0 according to the C standard §6.7.8 10:

> If an object that has static storage duration is not initialized explicitly, then:
— if it has pointer type, it is initialized to a null pointer;
— if it has arithmetic type, it is initialized to (positive or unsigned) zero;
— if it is an aggregate, every member is initialized (recursively) according to these rules;
— if it is a union, the first named member is initialized (recursively) according to these rules.

Reply:
Quite right!